linux/zynqlinux/custom_files/mount_storage.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/sh
-# mount second partition on SD card
-mount /dev/mmcblk0p2 /mnt/storage/
+# Mount second partition on SD card if not mounted yet.
+if ! grep -qs '/mnt/storage ' /proc/mounts; then
+    mount /dev/mmcblk0p2 /mnt/storage
```

Comment:
Here we need `mount -t ext2 /dev/mmcblk0p2 /mnt/storage`. Otherwise mount prints out a couple of warnings because it first tries to mount as ext4/ext3.